week1-section3/exercise3/Main.cpp

```diff
@@ -0,0 +1,83 @@
+/*Exercise 2 - More Input Validation
+Rewrite the true or false quiz you did in the previous section. This time, instead of asking for 1 or 2 as inputs, ask them to answer T or F. If they enter invalid input, give them a couple of tries before you automatically fail them.
+*/
+#include <iostream>
+
+
+int main()
+{
+	std::cout << "Welcome to this quiz\How many doy you know about general culture?"<<std::endl;
+	std::cout << "Remember to use F to false or T to true\nLet's play!!\n"<<std::endl;
+
+
+	//creating asks and answers
+	std::string question[5] = {"The paramecium is a flower? ","does the platypus lay eggs? ", "Is the whale a mammal? ","Is the bat a bird? ", "Is the tomatoe a fruit? "};
+	std::string answer[5] = {"F","T","T","F","T"};
+	
+	//creating inputs
+	std::string answer2;
+	int score=0;
+
+	int ansWrong=0;//to validate the wrong typing
+
+	for ( int i = 0; i < 5; i++)
```

Comment:
Let's discuss this exercise tomorrow :) It seems to be working fine for the most part, but it is a bit over-complicated.